### DISCUSSION TITLE
v4 Releases / Roadmap Question

### DISCUSSION THREAD

**Discussion by mandyMooreFan** [Akryum/vue-apollo#1498] [General]
Thanks for all the hard work. This project has been very helpful. I was skimming the repo and wasn't sure what else is needed for v4 to be moved out of beta.

**Comment by smolinari** [comment]
There are a number of issues that need attention. I wish I had the time and expertise and obviously @Akryum doesn't have the time (unfortunately). 

Seeing the popularity of this package (downloaded almost 80k per week) it really should get the attention it deserves. 

Scott

**Reply by mandyMooreFan** [reply]
Yeah I'd really like to contribute some time and energy to helping. I wasn't sure what issues were blocking the v4 release. The v4 tag was where I started, but wasn't sure what needed to be closed and what was a nice to have.
